options.css

```diff
@@ -0,0 +1,82 @@
+#filter-footer {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+#submit-button {
+    letter-spacing: 0.14em;
+    text-transform: uppercase;
+    font-size: 16px;
+    line-height: 16px;
+    margin-top: 24px;
+    outline: none;
+    padding: 10px 24px;
+    border: 0;
+    border-radius: 4px;
+    background-color: #f3f3f3;
+    color: #767676;
+    text-align: center;
+    cursor: pointer;
+}
+
+#submit-button:disabled {
+    opacity: 0.6;
+    cursor: not-allowed;
+}
+
+#filter-status {
+    font-size: 14px;
+    font-family: "Ideal Sans A", "Ideal Sans B", Helvetica, Arial, sans-serif;
+}
+
+.filter-option {
+    position: relative;
+    width: 100%;
+    text-align: left;
+    font-family: "Ideal Sans A", "Ideal Sans B", Helvetica, Arial, sans-serif;
+    padding-bottom: 16px;
+    margin-top: 10px;
+}
+
+.filter-option-header {
+    font: inherit;
+    vertical-align: baseline;
+    letter-spacing: 0.14em;
+    text-transform: uppercase;
+    font-size: 16px;
+    position: relative;
+    width: 100%;
+    text-align: left;
+    font-family: "Ideal Sans A", "Ideal Sans B", Helvetica, Arial, sans-serif;
+}
+
+.filter-option-description {
+    margin: 5px 0px 10px 0px;
+    font-size: 12px;
+    display: inline-block;
+    color: #767676;
+    text-align: center;
+    font-family: "Ideal Sans A", "Ideal Sans B", Helvetica, Arial, sans-serif;
+}
+
+.range-slider__custom-row {
```

Comment:
Given that the scope changed to make this not a range slider for the time being, should these classes be renamed?